README.md
# DripiIcons Icon
DripIcons Flutter 

#### Thanks to authors of Dripicons [Amit Jakhu](https://github.com/amitjakhu/dripicons)

##### DripiIcons 
![alt text](https://raw.githubusercontent.com/m-noer/dripicons_flutter/master/assets/dripicons.png "Dripicons")

##### DripiIcons Weather 
![alt text](https://raw.githubusercontent.com/m-noer/dripicons_flutter/master/assets/dripicons_weather.png "Dripicons weather")

## Usage
```dart
class MyHomePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Column(
        children: [
          Icon(
            DripIcons.alarm,
            size: 100,
            color: Colors.blue,
          ),
          Icon(
            DripIconsWeather.cloud_2,
            size: 100,
            color: Colors.blue,
          ),
        ],
      ),
    );
  }
}
```
## Example

```dart
import 'package:flutter/material.dart';
import 'package:windows_icon/windows_icon.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'DripIcons',
      theme: ThemeData(
        primarySwatch: Colors.blue,
      ),
      home: MyHomePage(),
    );
  }
}

class MyHomePage extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return Scaffold(
      body: Column(
        children: [
          Icon(
            DripIcons.alarm,
            size: 100,
            color: Colors.blue,
          ),
          Icon(
            DripIconsWeather.cloud_2,
            size: 100,
            color: Colors.blue,
          ),
        ],
      ),
    );
  }
}
```

CHANGELOG.md
## [0.0.1] - (02-09-2021)

* Initial release


test/dripicons_flutter_test.dart
import 'package:flutter_test/flutter_test.dart';

import 'package:dripicons_flutter/dripicons_flutter.dart';

void main() {
  test('DripIcons', () {
    expect(DripIcons.alarm.codePoint, 0x0061);
    expect(DripIcons.alarm.fontFamily, "DripIcons");
    expect(DripIcons.alarm.fontPackage, "dripicons_flutter");
  });
  test('DripIconsWeather', () {
    expect(DripIconsWeather.cloud_2.codePoint, 0x0061);
    expect(DripIconsWeather.cloud_2.fontFamily, "DripIconsWeather");
    expect(DripIconsWeather.cloud_2.fontPackage, "dripicons_flutter");
  });
}


lib/dripicons_weather.dart
part of dripicons_flutter;

class DripIconsWeather {
  /// DripIcons Weather

  /// ![cloud_2]
  static const IconData cloud_2 = DripIconsWeatherData(0x0061);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle = DripIconsWeatherData(0x0062);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_lightning = DripIconsWeatherData(0x0063);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_lightning_moon =
      DripIconsWeatherData(0x0064);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_lightning_sun =
      DripIconsWeatherData(0x0065);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_lightning_sun_2 =
      DripIconsWeatherData(0x0066);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_moon = DripIconsWeatherData(0x0067);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_sun = DripIconsWeatherData(0x0068);

  /// ![DripIconsWeatherData]
  static const IconData cloud_drizzle_sun_2 = DripIconsWeatherData(0x0069);

  /// ![DripIconsWeatherData]
  static const IconData cloud_fog = DripIconsWeatherData(0x006a);

  /// ![DripIconsWeatherData]
  static const IconData cloud_fog_2 = DripIconsWeatherData(0x006b);

  /// ![DripIconsWeatherData]
  static const IconData cloud_fog_moon = DripIconsWeatherData(0x006c);

  /// ![DripIconsWeatherData]
  static const IconData cloud_fog_sun = DripIconsWeatherData(0x006d);

  /// ![DripIconsWeatherData]
  static const IconData cloud_fog_sun_2 = DripIconsWeatherData(0x006e);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail = DripIconsWeatherData(0x006f);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_lightning = DripIconsWeatherData(0x0070);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_lightning_moon =
      DripIconsWeatherData(0x0071);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_lightning_sun = DripIconsWeatherData(0x0072);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_lightning_sun_2 =
      DripIconsWeatherData(0x0073);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_moon = DripIconsWeatherData(0x0074);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_sun = DripIconsWeatherData(0x0075);

  /// ![DripIconsWeatherData]
  static const IconData cloud_hail_sun_2 = DripIconsWeatherData(0x0076);

  /// ![DripIconsWeatherData]
  static const IconData cloud_lightning = DripIconsWeatherData(0x0077);

  /// ![DripIconsWeatherData]
  static const IconData cloud_lightning_moon = DripIconsWeatherData(0x0078);

  /// ![DripIconsWeatherData]
  static const IconData cloud_lightning_sun = DripIconsWeatherData(0x0079);

  /// ![DripIconsWeatherData]
  static const IconData cloud_lightning_sun_2 = DripIconsWeatherData(0x007a);

  /// ![DripIconsWeatherData]
  static const IconData cloud_moon = DripIconsWeatherData(0x0041);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain = DripIconsWeatherData(0x0042);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_2 = DripIconsWeatherData(0x0043);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_2_moon = DripIconsWeatherData(0x0044);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_2_sun = DripIconsWeatherData(0x0045);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_2_sun_2 = DripIconsWeatherData(0x0046);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_lightning = DripIconsWeatherData(0x0047);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_lightning_moon =
      DripIconsWeatherData(0x0048);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_lightning_sun = DripIconsWeatherData(0x0049);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_lightning_sun_2 =
      DripIconsWeatherData(0x004a);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_moon = DripIconsWeatherData(0x004b);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_sun = DripIconsWeatherData(0x004c);

  /// ![DripIconsWeatherData]
  static const IconData cloud_rain_sun_2 = DripIconsWeatherData(0x004d);

  /// ![DripIconsWeatherData]
  static const IconData cloud_snow = DripIconsWeatherData(0x004e);

  /// ![DripIconsWeatherData]
  static const IconData cloud_snow_moon = DripIconsWeatherData(0x004f);

  /// ![DripIconsWeatherData]
  static const IconData cloud_snow_sun = DripIconsWeatherData(0x0050);

  /// ![DripIconsWeatherData]
  static const IconData cloud_snow_sun_2 = DripIconsWeatherData(0x0051);

  /// ![DripIconsWeatherData]
  static const IconData cloud_sun = DripIconsWeatherData(0x0052);

  /// ![DripIconsWeatherData]
  static const IconData cloud_sun_2 = DripIconsWeatherData(0x0053);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind = DripIconsWeatherData(0x0054);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_2 = DripIconsWeatherData(0x0055);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_2_moon = DripIconsWeatherData(0x0056);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_2_sun = DripIconsWeatherData(0x0057);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_2_sun_2 = DripIconsWeatherData(0x0058);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_moon = DripIconsWeatherData(0x0059);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_sun = DripIconsWeatherData(0x005a);

  /// ![DripIconsWeatherData]
  static const IconData cloud_wind_sun_2 = DripIconsWeatherData(0x0030);

  /// ![DripIconsWeatherData]
  static const IconData clouds = DripIconsWeatherData(0x0031);

  /// ![DripIconsWeatherData]
  static const IconData clouds_moon = DripIconsWeatherData(0x0032);

  /// ![DripIconsWeatherData]
  static const IconData clouds_sun = DripIconsWeatherData(0x0033);

  /// ![DripIconsWeatherData]
  static const IconData clouds_sun_2 = DripIconsWeatherData(0x0034);

  /// ![DripIconsWeatherData]
  static const IconData compass_east = DripIconsWeatherData(0x0035);

  /// ![DripIconsWeatherData]
  static const IconData compass_north = DripIconsWeatherData(0x0036);

  /// ![DripIconsWeatherData]
  static const IconData compass_south = DripIconsWeatherData(0x0037);

  /// ![DripIconsWeatherData]
  static const IconData compass_west = DripIconsWeatherData(0x0038);

  /// ![DripIconsWeatherData]
  static const IconData degrees_celcius = DripIconsWeatherData(0x0039);

  /// ![DripIconsWeatherData]
  static const IconData degrees_fahrenheit = DripIconsWeatherData(0x0021);

  /// ![DripIconsWeatherData]
  static const IconData drizzle = DripIconsWeatherData(0x0022);

  /// ![DripIconsWeatherData]
  static const IconData flag = DripIconsWeatherData(0x0023);

  /// ![DripIconsWeatherData]
  static const IconData fog = DripIconsWeatherData(0x0024);

  /// ![DripIconsWeatherData]
  static const IconData hail = DripIconsWeatherData(0x0025);

  /// ![DripIconsWeatherData]
  static const IconData lightning = DripIconsWeatherData(0x0026);

  /// ![DripIconsWeatherData]
  static const IconData moon_25 = DripIconsWeatherData(0x0027);

  /// ![DripIconsWeatherData]
  static const IconData moon_50 = DripIconsWeatherData(0x0028);

  /// ![DripIconsWeatherData]
  static const IconData moon_75 = DripIconsWeatherData(0x0029);

  /// ![DripIconsWeatherData]
  static const IconData moon_100 = DripIconsWeatherData(0x002a);

  /// ![DripIconsWeatherData]
  static const IconData moon_stars = DripIconsWeatherData(0x002b);

  /// ![DripIconsWeatherData]
  static const IconData rain = DripIconsWeatherData(0x002c);

  /// ![DripIconsWeatherData]
  static const IconData raindrop = DripIconsWeatherData(0x002d);

  /// ![DripIconsWeatherData]
  static const IconData snow = DripIconsWeatherData(0x002e);

  /// ![DripIconsWeatherData]
  static const IconData sun = DripIconsWeatherData(0x002f);

  /// ![DripIconsWeatherData]
  static const IconData sun_eclipse = DripIconsWeatherData(0x003a);

  /// ![DripIconsWeatherData]
  static const IconData sun_low = DripIconsWeatherData(0x003b);

  /// ![DripIconsWeatherData]
  static const IconData sun_lower = DripIconsWeatherData(0x003c);

  /// ![DripIconsWeatherData]
  static const IconData sun_rise = DripIconsWeatherData(0x003d);

  /// ![DripIconsWeatherData]
  static const IconData sun_set = DripIconsWeatherData(0x003e);

  /// ![DripIconsWeatherData]
  static const IconData thermometer_25 = DripIconsWeatherData(0x003f);

  /// ![DripIconsWeatherData]
  static const IconData thermometer_50 = DripIconsWeatherData(0x0040);

  /// ![DripIconsWeatherData]
  static const IconData thermometer_75 = DripIconsWeatherData(0x005b);

  /// ![DripIconsWeatherData]
  static const IconData thermometer_100 = DripIconsWeatherData(0x005d);

  /// ![DripIconsWeatherData]
  static const IconData tornado = DripIconsWeatherData(0x005e);

  /// ![DripIconsWeatherData]
  static const IconData umbrella = DripIconsWeatherData(0x005f);

  /// ![DripIconsWeatherData]
  static const IconData umbrella_drizzle = DripIconsWeatherData(0x0060);

  /// ![DripIconsWeatherData]
  static const IconData wet = DripIconsWeatherData(0x007b);

  /// ![DripIconsWeatherData]
  static const IconData wind = DripIconsWeatherData(0x007c);
}


lib/dripicons.dart
part of dripicons_flutter;

class DripIcons {
  static const IconData alarm = DripIconsData(0x0061);
  static const IconData align_center = DripIconsData(0x0062);
  static const IconData align_justify = DripIconsData(0x0063);
  static const IconData align_left = DripIconsData(0x0064);
  static const IconData align_right = DripIconsData(0x0065);
  static const IconData anchor = DripIconsData(0x0066);
  static const IconData archive = DripIconsData(0x0067);
  static const IconData arrow_down = DripIconsData(0x0068);
  static const IconData arrow_left = DripIconsData(0x0069);
  static const IconData arrow_right = DripIconsData(0x006a);
  static const IconData arrow_thin_down = DripIconsData(0x006b);
  static const IconData arrow_thin_left = DripIconsData(0x006c);
  static const IconData arrow_thin_right = DripIconsData(0x006d);
  static const IconData arrow_thin_up = DripIconsData(0x006e);
  static const IconData arrow_up = DripIconsData(0x006f);
  static const IconData article = DripIconsData(0x0070);
  static const IconData backspace = DripIconsData(0x0071);
  static const IconData basket = DripIconsData(0x0072);
  static const IconData basketball = DripIconsData(0x0073);
  static const IconData battery_empty = DripIconsData(0x0074);
  static const IconData battery_full = DripIconsData(0x0075);
  static const IconData battery_low = DripIconsData(0x0076);
  static const IconData battery_medium = DripIconsData(0x0077);
  static const IconData bell = DripIconsData(0x0078);
  static const IconData blog = DripIconsData(0x0079);
  static const IconData bluetooth = DripIconsData(0x007a);
  static const IconData bold = DripIconsData(0x0041);
  static const IconData bookmark = DripIconsData(0x0042);
  static const IconData bookmarks = DripIconsData(0x0043);
  static const IconData box = DripIconsData(0x0044);
  static const IconData briefcase = DripIconsData(0x0045);
  static const IconData brightness_low = DripIconsData(0x0046);
  static const IconData brightness_max = DripIconsData(0x0047);
  static const IconData brightness_med = DripIconsData(0x0048);
  static const IconData broadcast = DripIconsData(0x0049);
  static const IconData browser = DripIconsData(0x004a);
  static const IconData browser_upload = DripIconsData(0x004b);
  static const IconData brush = DripIconsData(0x004c);
  static const IconData calendar = DripIconsData(0x004d);
  static const IconData camcorder = DripIconsData(0x004e);
  static const IconData camera = DripIconsData(0x004f);
  static const IconData card = DripIconsData(0x0050);
  static const IconData cart = DripIconsData(0x0051);
  static const IconData checklist = DripIconsData(0x0052);
  static const IconData checkmark = DripIconsData(0x0053);
  static const IconData chevron_down = DripIconsData(0x0054);
  static const IconData chevron_left = DripIconsData(0x0055);
  static const IconData chevron_right = DripIconsData(0x0056);
  static const IconData chevron_up = DripIconsData(0x0057);
  static const IconData clipboard = DripIconsData(0x0058);
  static const IconData clock = DripIconsData(0x0059);
  static const IconData clockwise = DripIconsData(0x005a);
  static const IconData cloud = DripIconsData(0x0030);
  static const IconData cloud_download = DripIconsData(0x0031);
  static const IconData cloud_upload = DripIconsData(0x0032);
  static const IconData code = DripIconsData(0x0033);
  static const IconData contract = DripIconsData(0x0034);
  static const IconData contract2 = DripIconsData(0x0035);
  static const IconData conversation = DripIconsData(0x0036);
  static const IconData copy = DripIconsData(0x0037);
  static const IconData crop = DripIconsData(0x0038);
  static const IconData cross = DripIconsData(0x0039);
  static const IconData crosshair = DripIconsData(0x0021);
  static const IconData cutlery = DripIconsData(0x0022);
  static const IconData device_desktop = DripIconsData(0x0023);
  static const IconData device_mobile = DripIconsData(0x0024);
  static const IconData device_tablet = DripIconsData(0x0025);
  static const IconData direction = DripIconsData(0x0026);
  static const IconData disc = DripIconsData(0x0027);
  static const IconData document = DripIconsData(0x0028);
  static const IconData document_delete = DripIconsData(0x0029);
  static const IconData document_edit = DripIconsData(0x002a);
  static const IconData document_new = DripIconsData(0x002b);
  static const IconData document_remove = DripIconsData(0x002c);
  static const IconData dot = DripIconsData(0x002d);
  static const IconData dots_2 = DripIconsData(0x002e);
  static const IconData dots_3 = DripIconsData(0x002f);
  static const IconData download = DripIconsData(0x003a);
  static const IconData duplicate = DripIconsData(0x003b);
  static const IconData enter = DripIconsData(0x003c);
  static const IconData exit = DripIconsData(0x003d);
  static const IconData expand = DripIconsData(0x003e);
  static const IconData expand_2 = DripIconsData(0x003f);
  static const IconData experiment = DripIconsData(0x0040);
  static const IconData export = DripIconsData(0x005b);
  static const IconData feed = DripIconsData(0x005e);
  static const IconData flag = DripIconsData(0x005f);
  static const IconData flashlight = DripIconsData(0x0060);
  static const IconData folder = DripIconsData(0x007b);
  static const IconData folder_open = DripIconsData(0x007c);
  static const IconData fowrward = DripIconsData(0x007d);
  static const IconData gaming = DripIconsData(0x007e);
  static const IconData gear = DripIconsData(0x005c);
  static const IconData graduation = DripIconsData(0xe01a);
  static const IconData graph_bar = DripIconsData(0xe000);
  static const IconData graph_line = DripIconsData(0xe001);
  static const IconData graph_lie = DripIconsData(0xe002);
  static const IconData headset = DripIconsData(0xe003);
  static const IconData heart = DripIconsData(0xe004);
  static const IconData help = DripIconsData(0xe005);
  static const IconData home = DripIconsData(0xe006);
  static const IconData hourglass = DripIconsData(0xe007);
  static const IconData inbox = DripIconsData(0xe008);
  static const IconData information = DripIconsData(0xe009);
  static const IconData italic = DripIconsData(0xe00a);
  static const IconData jewel = DripIconsData(0xe00b);
  static const IconData lifting = DripIconsData(0xe00c);
  static const IconData lightbulb = DripIconsData(0xe00d);
  static const IconData link = DripIconsData(0xe00e);
  static const IconData link_broken = DripIconsData(0xe00f);
  static const IconData list = DripIconsData(0xe010);
  static const IconData loading = DripIconsData(0xe011);
  static const IconData location = DripIconsData(0xe012);
  static const IconData lock = DripIconsData(0xe013);
  static const IconData lockOpen = DripIconsData(0xe014);
  static const IconData mail = DripIconsData(0xe015);
  static const IconData map = DripIconsData(0xe016);
  static const IconData media_loop = DripIconsData(0xe017);
  static const IconData media_next = DripIconsData(0xe018);
  static const IconData media_pause = DripIconsData(0xe019);
  static const IconData media_play = DripIconsData(0xe01a);
  static const IconData media_previous = DripIconsData(0xe01b);
  static const IconData media_record = DripIconsData(0xe01c);
  static const IconData media_shuffle = DripIconsData(0xe01d);
  static const IconData media_stop = DripIconsData(0xe01e);
  static const IconData medical = DripIconsData(0xe01f);
  static const IconData menu = DripIconsData(0xe020);
  static const IconData message = DripIconsData(0xe021);
  static const IconData meter = DripIconsData(0xe022);
  static const IconData microphone = DripIconsData(0xe023);
  static const IconData minus = DripIconsData(0xe024);
  static const IconData monitor = DripIconsData(0xe025);
  static const IconData move = DripIconsData(0xe026);
  static const IconData music = DripIconsData(0xe027);
  static const IconData network1 = DripIconsData(0xe028);
  static const IconData network2 = DripIconsData(0xe029);
  static const IconData network3 = DripIconsData(0xe02a);
  static const IconData network4 = DripIconsData(0xe02b);
  static const IconData network5 = DripIconsData(0xe02c);
  static const IconData pamphlet = DripIconsData(0xe02d);
  static const IconData paperclip = DripIconsData(0xe02e);
  static const IconData pencil = DripIconsData(0xe02f);
  static const IconData phone = DripIconsData(0xe030);
  static const IconData photo = DripIconsData(0xe031);
  static const IconData photoGroup = DripIconsData(0xe032);
  static const IconData pill = DripIconsData(0xe033);
  static const IconData pin = DripIconsData(0xe034);
  static const IconData plus = DripIconsData(0xe035);
  static const IconData power = DripIconsData(0xe036);
  static const IconData preview = DripIconsData(0xe037);
  static const IconData print = DripIconsData(0xe038);
  static const IconData pulse = DripIconsData(0xe039);
  static const IconData question = DripIconsData(0xe03a);
  static const IconData reply = DripIconsData(0xe03b);
  static const IconData reply_all = DripIconsData(0xe03c);
  static const IconData return_icon = DripIconsData(0xe03d);
  static const IconData retweet = DripIconsData(0xe03e);
  static const IconData rocket = DripIconsData(0xe03f);
  static const IconData scale = DripIconsData(0xe040);
  static const IconData search = DripIconsData(0xe041);
  static const IconData shopping_bag = DripIconsData(0xe042);
  static const IconData skip = DripIconsData(0xe043);
  static const IconData stack = DripIconsData(0xe044);
  static const IconData star = DripIconsData(0xe045);
  static const IconData stopwatch = DripIconsData(0xe046);
  static const IconData store = DripIconsData(0xe047);
  static const IconData suitcase = DripIconsData(0xe048);
  static const IconData swap = DripIconsData(0xe049);
  static const IconData tag = DripIconsData(0xe04a);
  static const IconData tag_delete = DripIconsData(0xe04b);
  static const IconData tags = DripIconsData(0xe04c);
  static const IconData thumbs_down = DripIconsData(0xe04d);
  static const IconData thumbs_up = DripIconsData(0xe04e);
  static const IconData ticket = DripIconsData(0xe04f);
  static const IconData time_reverse = DripIconsData(0xe050);
  static const IconData to_do = DripIconsData(0xe051);
  static const IconData toggles = DripIconsData(0xe052);
  static const IconData trash = DripIconsData(0xe053);
  static const IconData trophy = DripIconsData(0xe054);
  static const IconData upload = DripIconsData(0xe055);
  static const IconData user = DripIconsData(0xe056);
  static const IconData user_group = DripIconsData(0xe057);
  static const IconData user_id = DripIconsData(0xe058);
  static const IconData vibrate = DripIconsData(0xe059);
  static const IconData view_apps = DripIconsData(0xe05a);
  static const IconData view_list = DripIconsData(0xe05b);
  static const IconData view_list_page = DripIconsData(0xe05c);
  static const IconData view_thumb = DripIconsData(0xe05d);
  static const IconData volume_full = DripIconsData(0xe05e);
  static const IconData volume_low = DripIconsData(0xe05f);
  static const IconData volume_medium = DripIconsData(0xe060);
  static const IconData volume_off = DripIconsData(0xe061);
  static const IconData wallet = DripIconsData(0xe062);
  static const IconData warning = DripIconsData(0xe063);
  static const IconData web = DripIconsData(0xe064);
  static const IconData weight = DripIconsData(0xe065);
  static const IconData wifi = DripIconsData(0xe066);
  static const IconData wrong = DripIconsData(0xe067);
  static const IconData zoom_in = DripIconsData(0xe068);
  static const IconData zoom_out = DripIconsData(0xe069);
}
